requirements/common.in

```diff
@@ -3,110 +3,51 @@ gunicorn==20.0.4
 whitenoise[brotli]==5.0.1
 
 # Used by Whitenoise to provide Brotli-compressed versions of static files.
-Brotli==1.0.7
-Django==3.0.3
+Django==3.0.5
 celery==4.3.0
-kombu==4.6.3
 simplejson==3.17.0
 newrelic==5.2.1.129
 
 # Required by Django
 mysqlclient==1.4.6
-asgiref==3.2.3
-
-# Required by celery
-billiard==3.6.2.0
-pytz==2019.3
-
-# Required by kombu
-amqp==2.5.2
-
-# Required by amqp
-vine==1.3.0
 
 # Required by mozlog
-mozterm==1.0.0
-blessings==1.7
 jsonschema==3.0.1
 djangorestframework==3.11.0
 django-cors-headers==3.2.1
 mozlog==5.0
 
 # Required by djangorestframework's API documentation
 coreapi==2.3.3
-coreschema==0.0.4
-Jinja2==2.11.1
-MarkupSafe==1.1.1
-
-# Required by coreapi
-itypes==1.1.0
-uritemplate==3.0.1
-requests==2.22.0
-
-# Required by django.contrib.migrations
-sqlparse==0.3.0
 
 # Used directly and also by Django's YAML serializer.
 PyYAML==5.3
 django-environ==0.4.5
 
 # Whilst we don't use six ourselves any more, it's still required by a number of dependencies.
-six==1.14.0
-python-dateutil==2.8.1
-django-filter==2.2.0
-django-redis==4.11.0
-
-# Required by django-redis
-redis==3.4.1
-elasticsearch==7.5.1
-
-# required by requests and elasticsearch
-urllib3==1.25.8
-certifi==2019.11.28
-
-# required by requests
-chardet==3.0.4
-idna==2.8
+python-dateutil
```

Comment:
Ideally, we would remove version form all, but that causes test failures: Treeherder depends on some quirks of specific versions. The debug cycle for testing removal of a version number is very long; it is best to distribute that cost over other PRs; triggered by a bot to suggest it may be safe

Reply:
Sure.